src/main/interface/settings.c

```diff
@@ -594,6 +594,7 @@ const clivalue_t valueTable[] = {
 
 // PG_MIXER_CONFIG
     { "yaw_motors_reversed",        VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, yaw_motors_reversed) },
+    { "ddd_mixergain_halving",       VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, ddd_mixergain_halving) },
```

Comment:
The paramter name doesn't have to match the variable name so you can use a "3d_"... type of name.  How about `3d_mixer_gain_fix` ?

Reply:
Also even though I realize it's not consistently done (look at the item above as an example), the data type should be `VAR_UINT8` to be more correct.  In practice it doesn't really matter because of the `TABLE_ON_OFF` constraint but it still would be more correct.

Reply:
I didn't realise the two weren't one and the same. I'll rename the parameter, change the variable type and then update the variable name from the previous suggestion. This shines light on how it's actually working, thanks.